cmd/libs/go2idl/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -148,49 +152,85 @@ func (g *genFakeForType) GenerateType(c *generator.Context, t *types.Type, w io.
 		"ExtractFromListOptions":         c.Universe.Function(types.Name{Package: pkgClientGoTesting, Name: "ExtractFromListOptions"}),
 	}
 
-	noMethods := extractBoolTagOrDie("noMethods", t.SecondClosestCommentLines) == true
-
-	readonly := extractBoolTagOrDie("readonly", t.SecondClosestCommentLines) == true
+	if len(tags.OnlyVerbs) > 0 {
+		tags.SkipVerbs = []string{}
+		for _, m := range util.SupportedVerbs {
+			skip := true
+			for _, o := range tags.OnlyVerbs {
+				if o == m {
+					skip = false
+					break
+				}
+			}
+			if skip {
+				tags.SkipVerbs = append(tags.SkipVerbs, m)
+			}
+		}
+	}
 
-	if namespaced {
-		sw.Do(structNamespaced, m)
-	} else {
+	if tags.NonNamespaced {
 		sw.Do(structNonNamespaced, m)
+	} else {
+		sw.Do(structNamespaced, m)
 	}
 
-	if !noMethods {
+	if !tags.NoVerbs {
 		sw.Do(resource, m)
 		sw.Do(kind, m)
 	}
 
-	if !noMethods && !readonly {
-		sw.Do(createTemplate, m)
-		sw.Do(updateTemplate, m)
-		// Generate the UpdateStatus method if the type has a status
-		if genStatus(t) {
+	if !tags.NoVerbs && !tags.Readonly {
```

Comment:
After you generate getters, you can check if its readonly and just return, right?